### ISSUE TITLE
Relative path to test is broken? (w.r.t cram?)

### ISSUE THREAD

**Issue by nigels-com** [Nicoretti/prysk#107]
I had trouble running prysk via cmake.  It's using a rule as follows:

```
    add_test(
        NAME ${cram_test_name}
        COMMAND prysk --xunit-file=${CMAKE_BINARY_DIR}/${cram_test_name}.xml ${cram_test}
        WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
    )
```
I think the problem is that  ${cram_test} is a relative path...

```
$ /usr/local/bin/prysk --verbose --xunit-file foo.t.xml ../tmp/foo.t
no tests found
$ ls -lah ../tmp/foo.t
-rwxr-xr-x 1 nigels nigels 3.7K Jun 14 16:14 ../tmp/foo.t
```

Whereas an absolute path or local file works fine:
```
foo.t: passed
# Ran 1 tests, 0 skipped, 0 failed.
```
```

**Comment by Nicoretti** [comment]
Hi @nigels-com ,

Thanks for reporting the issue! I had a quick look at it and as far as I can tell your assumption, regarding the realtive path resolution seems to  be correct. I'll try to come up with a test case in the next days and then address the isssue.

best
Nico

**Comment by nigels-com** [comment]
Awesome, thanks!

**Comment by Nicoretti** [comment]
@nigels-com I just published `prysk 0.12.2` which should fix you issue, if it doesn't feel free to reopen the issue.

best
Nico

**Comment by nigels-com** [comment]
Yep, perfect!  Thanks!
